src/routes/klageskjema/begrunnelse/autosaveProgress.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { Element } from 'nav-frontend-typografi';
+import styled from 'styled-components/macro';
+import { Ellipsis } from '../../../styled-components/ellipsis';
+import Success from '../../../assets/images/icons/Success';
+import Popover, { PopoverOrientering } from 'nav-frontend-popover';
+
+export enum AutosaveStatus {
+    SAVING = 0,
+    SAVED = 1,
+    NOT_SAVED = 2
+}
+
+interface Props {
+    autosaveStatus: number | undefined;
```

Comment:
Bruk `AutosaveStatus` her også :)

Reply:
Kunne vi også lagt til en verdi i enumen i stedet for å bruke `undefined`?
Typ. `NONE` f.eks. (kommer ikke på noe bedre nå 😛 )?

Da setter vi den som initial value i `useState` i `Begrunnelse` og fjerner all bruk av `undefined`.